qt/scientific_interfaces/Indirect/IndirectFitData.cpp

```diff
@@ -212,6 +212,7 @@ Spectra::Spectra(WorkspaceIndex minimum, WorkspaceIndex maximum) {
   }
   m_vec.resize(maximum.value - minimum.value + 1);
   std::iota(m_vec.begin(), m_vec.end(), minimum);
+  m_isContinuous = true;
```

Comment:
May I ask why the removal of the legacy code requires additional changes in code that is being kept. Are they further bugfixes?

Reply:
There were some unit tests which tested the legacy code which I re-purposed to point at the new classes instead. This showed up a couple of issues which were corrected. In this case the value of m_isContinuous was not being set in this constructor. This was ok on most compilers as it defaulted to true but was failing the Ubuntu builds.

Reply:
So it the PR should probably be entitled remove legacy code and add unit tests.

Reply:
Ah lovely. Always nice to catch issues like this with tests. :+1: